Source/items.cpp

```diff
@@ -4300,11 +4300,11 @@ void UseItem(int p, item_misc_id Mid, spell_id spl)
 			plr[p]._pHPBase = plr[p]._pMaxHPBase;
 		drawhpflag = true;
 		j = plr[p]._pMaxMana >> 8;
-		l = ((j >> 1) + random_(40, j)) << 6;
+		l = ((j / 2) + random_(40, j)) << 6;
 		if (plr[p]._pClass == HeroClass::Sorcerer)
 			l <<= 1;

```

Comment:
Missed this?

Reply:
Hm no, I think you reviewed one commit instead of the whole PR.

Reply:
Ah sorry you are right, you were mentioning line 4305. I did miss that yes.